charts/karpenter/values.yaml

```diff
@@ -4,7 +4,11 @@ serviceAccount:
   # Annotations to add to the service account (like the ARN of the IRSA role)
   annotations: {}
 controller:
+  # List of environment items to add to the controller, for example
+  # - name: AWS_DEFAULT_REGION
```

Comment:
I think this variable should be AWS_REGION from what we've learned, now.

Reply:
Agreed.